packages/core/src/kernelCoreTemplate.js

```diff
@@ -59,7 +59,7 @@
     getRelativeModuleId,
     prepareModuleInitializerArgs,
     getExternalCompartment,
-    globalThisRefs = ['globalThis'],
+    globalThisRefs = ['window', 'self', 'global', 'globalThis', 'top', 'frames', 'parent'],
```

Comment:
(must) doublecheck - I think this is getting overriden whereever kernel is being used. I think this change should not be here.

Reply:
in that case we might be missing an override. this should not be included in lavamoat-node and should be specified by either lavapack or lm-browserify

Reply:
> Doesn't seem like it [b37a74d](https://github.com/LavaMoat/LavaMoat/commit/b37a74db7cfa074ed71e249ae0fd45be5995451d)

oh, maybe that's because this test belongs in lavapack or browserify or needs to itself be parametrized?

Reply:
I know what's the best way out - make this field mandatory and throw if not set - we'll quickly learn where it requires updating.

Reply:
addressed https://github.com/LavaMoat/LavaMoat/pull/1505#issuecomment-2619300114